app/controllers/defendants_controller.rb

```diff
@@ -47,6 +56,12 @@ def prosecution_case_reference
 
   private
 
+  def handle_link_failure(message, exception = nil)
+    logger.warn "LINK DEFENDANT FAILURE (params: #{@unlink_attempt.as_json}): #{message}"
```

Comment:
This should say `UNLINK DEFENDANT...` and the method should probably be called `handle_unlink_failure`